### PR TITLE
Distance sort

### PR DESCRIPTION
**Pull request by franktucci**
Basic sort and filter by distance/radius. Am continuing working on updating the homepage when a user sets the location to a new one

### CODE REVIEW

File: backend/models/listing.js [hunk 1]
```diff
@@ -23,13 +23,14 @@ const ListingSchema = new mongoose.Schema(
...
       address: {
         type: String,
-        required: true,
       },
-      latitude: {
-        type: Number,
-      },
-      longitude: {
-        type: Number,
+      latlng: {
```
Comment (KimLinhVu) [inline backend/models/listing.js:27]: If we change the listing model here, we need to make sure that all the other places in the code that make a listing object are updated as well.

**Review by KimLinhVu — APPROVED** [review]
LGTM.

File: backend/routes/listing.js [hunk 2]
```diff
@@ -149,23 +123,21 @@ async function getListings(
...
   if (claimed) {
-    match.push({ claimed: claimed });
+    if (claimed === "true") {
+      match.push({ claimed: true });
+    } else {
+      match.push({ claimed: false });
+    }
   } else {
     match.push({ claimed: false });
   }
```
Comment (vilinh) [inline backend/routes/listing.js]: Shouldn't we just not push a claimed filter if it is not set to true?
So
`
  if (claimed) {
    match.push({ claimed: true });
  }
`

Comment (franktucci) [reply backend/routes/listing.js]: Good idea!

File: backend/routes/listing.js [hunk 3]
```diff
@@ -145,6 +146,20 @@ async function getListings(
...
+  console.log(latlng);
+  console.log(sanitize(latlng));
+  console.log(sanitize(latlng).split(","));
+  try {
+    lat = sanitize(latlng).split(",")[0];
+    lng = sanitize(latlng).split(",")[1];
+  } catch {}
+
+  console.log(lat);
+  console.log(lng);
```
Comment (filip-conic) [inline backend/routes/listing.js]: nit: console logs should probably be deleted before commiting

**Review by filip-conic — COMMENTED** [review]
small nit, overall LGTM

File: backend/routes/listing.js (hunk 2 above)
Comment (filip-conic) [reply backend/routes/listing.js]: if, else isn't necessary here. `match.push({ claimed: claimed })` does the same thing

**Pull request merged**